components/support/nimbus-fml/src/workflows.rs

```diff
@@ -180,17 +180,46 @@ mod test {
         })
     }
 
+    fn generate_multiple_and_assert(
+        test_script: &str,
+        manifests: &[&str],
+        channel: &str,
+    ) -> Result<()> {
+        let cmds = manifests.iter().map(|manifest| {
+            let cmd = create_command_from_test(test_script, manifest, channel, false)?;
+            generate_struct(&cmd)?;
+
+            Ok(cmd)
+        });
```

Comment:
nit: can we add the `collect::<Result<Vec<_>>>()?;` here and possibly cut the next two lines somehow?